class/defaults.yml

```diff
@@ -1,3 +1,7 @@
 parameters:
   eventrouter:
     namespace: syn-eventrouter
+    images:
+      eventrouter:
+        image: "gcr.io/heptio-images/eventrouter"
```

Comment:
I'd suggest we split this further with a `registry` variable to be able to override only the registry part (see [example](https://github.com/projectsyn/component-thanos/blob/master/class/defaults.yml#L26))